lib/game.rb

```diff
@@ -0,0 +1,92 @@
+class Game
+  @@v_conditions = [[1,2,3], [4,5,6], [7,8,9], [1,4,7], [2,5,8], [3,6,9], [1,5,9], [3,5,7]]
+  attr_reader :board
+
+  def initialize(p1, p2)
+    @p1 = p1
+    @p2 = p2
+    @board = Board.new
```

Comment:
Why don't you just pass an instantiated board object as an argument like you did for p1 and p2?